install/jsonb/bin/ts.jte

```diff
@@ -50,8 +50,8 @@ jsonb.classes=
 #
 #                                Added to ts.run.classpath below.
 ###############################################################################
-#jsonb.resources=${ts.home}/src/com/sun/ts/tests/jsonb/resources
```

Comment:
the changes to install/jsonb/bin/ts.jte and install/jsonb/bin/build.xml are not required, these files will be removed as part of standalone tck remove PR(https://github.com/eclipse-ee4j/jakartaee-tck/pull/848)